engine/runtime-parser/src/main/java/org/enso/compiler/core/ir/Empty.java

```diff
@@ -0,0 +1,16 @@
+package org.enso.compiler.core.ir;
+
+import org.enso.runtime.parser.dsl.GenerateFields;
+import org.enso.runtime.parser.dsl.GenerateIR;
+
+@GenerateIR(interfaces = "org.enso.compiler.core.ir.Expression")
```

Comment:
Using `Class[]` might be better. Then this line would become:
```suggestion
@GenerateIR(interfaces =Expression.class)
```

Reply:
Already implemented. See https://github.com/enso-org/enso/pull/11770#discussion_r1899542105